src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -31,17 +31,24 @@ const propTypes = {
 
     /** From Onyx */
     reimbursementAccount: reimbursementAccountPropTypes,
+
+    /** User Data from Onyx */
     user: userPropTypes,
 
+    /** Main content of the page */
     children: PropTypes.func,
 
+    /** Content to be added as fixed footer */
+    footer: PropTypes.element,
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     children: () => {},
     user: {},
     reimbursementAccount: {},
+    footer: null,
```

Comment:
NAB; I think it could be nice to alphabetize these props or put them in the order they appear in `propTypes`. What do you think?